metrics/prometheus.go

```diff
@@ -33,9 +33,14 @@ import (
 // asFloat64 converts a uint64 into a float64.
 func asFloat64(v uint64) float64 { return float64(v) }
 
+// asMicrosecondsToSeconds converts nanoseconds into a float64 representing seconds.
+func asMicrosecondsToSeconds(v uint64) float64 {
+	return float64(v) / 1e6
+}
+
 // asNanosecondsToSeconds converts nanoseconds into a float64 representing seconds.
 func asNanosecondsToSeconds(v uint64) float64 {
-	return float64(v) / float64(time.Second)
```

Comment:
the old way reads as slightly easier to understand to me. What motivates this change?

Reply:
As mentioned in the review comments, it's confusing when you compare `asMicrosecondsToSeconds()`. The base unit is `time.Nanosecond`, so you would have to use `time.Millisecond` to convert microseconds to seconds.

By simply using the float factor, it's consistent to read both functions.